src/adapters/sqliteAdapter.ts

```diff
@@ -12,31 +11,20 @@ const prismaBinary = './node_modules/.bin/prisma'
 
 export function getConnectionString(databaseCredentials: EnvironmentDatabaseCredentials) {
   const { dbName, dbSchema } = databaseCredentials
-  return `file:./${dbName.replace('.db', '')}_${dbSchema}.db`
+  return `file:/tmp/${dbName.replace('.db', '')}_${dbSchema}.db`
```

Comment:
Have you tried this approach on non-linux environments? I think this may be crashing on Windows

Reply:
Good point, this will crash on Windows. I'll update to use `os.tmpdir()`.